### PR TITLE
changing design template

### PR DESCRIPTION
**Pull request by smarzban**

### CODE REVIEW

**Comment by smarzban** [comment]
@miltontony and @smn this is what I've done so far, ready for review.

**Comment by saeedmb8** [comment]
@smn ready for review :)

File: unicorecmsunfpa/templates/content.pt [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
                     </condition>
-                    <div class="h2">${page.title}</div>
                     <div class="meta">${page.description}</div>
                 </div>
                 <div class="clr"></div>
 
                 <p>${structure:content}</p>
-                <div class="date">${view.format_date(page.created_at)}</div>
-                <div class="items">
+<!--                 <div class="date">${view.format_date(page.created_at)}</div>
```
Comment (miltontony) [inline unicorecmsunfpa/templates/content.pt]: Rather just delete this since it's not likely to be put back again

**Comment by miltontony** [comment]
:+1: Just a general comment that rather than commenting things out, just delete them. No reason to keep around dead code.

**Comment by smarzban** [comment]
@miltontony Noted, thanks

**Comment by smarzban** [comment]
@miltontony and @smn ready for review please :)

**Comment by miltontony** [comment]
:+1:  let's land this

**Pull request merged**